lib/f_core/Kconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 RIT Launch Initiative
+# SPDX-License-Identifier: Apache-2.0
+
+config F_CORE
```

Comment:
As we get some more real subsystems in FSW we probably wanna get some readmes about structure or start using the github wiki to explain some of this stuff.

Reply:
yes

Reply:
Added a README in F-Core